src/i18n/translations.ts

```diff
@@ -31,12 +35,29 @@ export default {
         metrics: 'Metriken',
         logs: 'Logs',
         configuration: 'Einstellungen',
-        global: 'Global'
+        global: 'Global',
+        image:' Bilddateien'
```

Comment:
```suggestion
        image: 'Bilddateien'
```
also maybe a key such as `imageFiles` fits better